lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -18,7 +18,12 @@ def callback(request, _uri, headers):
         data along with the data from response_overrides and dummy values for any
         additional required fields.
         """
+        is_new_thread = thread_data.pop("new_thread")
         response_data = make_minimal_cs_thread(thread_data)
+        # pop resp_total as it is not part of response from CS on thread creation
+        # request
+        if is_new_thread:
+            response_data.pop("resp_total")
```

Comment:
No concern as long as the change doesn't introduce any performance regression and doesn't break the web.  :)